Services/Certificate/classes/Template/Action/Export/class.ilCertificateTemplateExportAction.php

```diff
@@ -97,20 +97,28 @@ public function export($rootDir = CLIENT_WEB_DIR, $installationId = IL_INST_ID)
 
         $backgroundImagePath = $template->getBackgroundImagePath();
         if ($backgroundImagePath !== null && $backgroundImagePath !== '') {
-            $this->filesystem->copy($backgroundImagePath, $exportPath . 'background.jpg');
+            try {
+                $this->filesystem->copy($backgroundImagePath, $exportPath . 'background.jpg');
+            } catch (\ILIAS\Filesystem\Exception\FileAlreadyExistsException $e) {
+            } catch (\ILIAS\Filesystem\Exception\FileNotFoundException $e) {
```

Comment:
Please add some logging here (use the `cert` logger `$DIC->logger()->cert()`, not the `root` logger).

Please use one `catch` block for the three exception types (use the `|`) for `trunk` (available for PHP >= 7.1, so please do **not** use this for `release_5-4`).